megalinter/descriptors/php.megalinter-descriptor.yml

```diff
@@ -135,8 +148,11 @@ linters:
       - "psalm --config=psalm.xml myfile.php mydir/"
     install:
       dockerfile:
+        - |-
+          # renovate: datasource=composer depName=vimeo/psalm
+          ARG PHP_VIMEO_PSALM_VERSION=6.8.8
         - |
-          RUN GITHUB_AUTH_TOKEN="$(cat /run/secrets/GITHUB_TOKEN)" && export GITHUB_AUTH_TOKEN && composer config --global platform.php 8.3 && composer global require vimeo/psalm
+          RUN GITHUB_AUTH_TOKEN="$(cat /run/secrets/GITHUB_TOKEN)" && export GITHUB_AUTH_TOKEN && composer config --global platform.php 8.4 && composer global require vimeo/psalm:${PHP_VIMEO_PSALM_VERSION}
```

Comment:
Just to note that changing php version might have a bigger impact on users if they aren't already following the depreciations

Reply:
I have left it as it was, I see that MegaLinter 8.4.2 uses 6.5.0 which is the last one compatible with PHP 8.3.0.